doc/source/tune/tutorials/tune_get_data_in_and_out.md

```diff
@@ -0,0 +1,516 @@
+      <th>warmup_time</th>
+      <th>config/epochs</th>
+      <th>config/hyperparameter_a</th>
+      <th>config/hyperparameter_b</th>
+      <th>logdir</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>-58.399962</td>
+      <td>1.015951</td>
+      <td>True</td>
+      <td>False</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>10</td>
+      <td>0b239_00000</td>
+      <td>acf38c19d59c4cf2ad7955807657b6ea</td>
+      <td>2022-11-30_17-40-26</td>
+      <td>...</td>
+      <td>ip-172-31-43-110</td>
+      <td>172.31.43.110</td>
+      <td>10.282120</td>
+      <td>0</td>
+      <td>10</td>
+      <td>0.003541</td>
+      <td>10</td>
+      <td>18.065981</td>
+      <td>-98.298928</td>
+      <td>/home/ubuntu/ray_results/training_function_202...</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>-24.461518</td>
+      <td>1.030420</td>
+      <td>True</td>
+      <td>False</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>10</td>
+      <td>0b239_00001</td>
+      <td>5ca9e03d7cca46a7852cd501bc3f7b38</td>
+      <td>2022-11-30_17-40-28</td>
+      <td>...</td>
+      <td>ip-172-31-43-110</td>
+      <td>172.31.43.110</td>
+      <td>10.362581</td>
+      <td>0</td>
+      <td>10</td>
+      <td>0.004031</td>
+      <td>10</td>
+      <td>1.544918</td>
+      <td>-47.741455</td>
+      <td>/home/ubuntu/ray_results/training_function_202...</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>18.510299</td>
+      <td>1.034228</td>
+      <td>True</td>
+      <td>False</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>10</td>
+      <td>0b239_00002</td>
+      <td>aa38dd786c714486a8d69fa5b372df48</td>
+      <td>2022-11-30_17-40-28</td>
+      <td>...</td>
+      <td>ip-172-31-43-110</td>
+      <td>172.31.43.110</td>
+      <td>10.333781</td>
+      <td>0</td>
+      <td>10</td>
+      <td>0.005286</td>
+      <td>10</td>
+      <td>8.129285</td>
+      <td>28.846415</td>
+      <td>/home/ubuntu/ray_results/training_function_202...</td>
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>-16.138780</td>
+      <td>1.020072</td>
+      <td>True</td>
+      <td>False</td>
+      <td>NaN</td>
+      <td>NaN</td>
+      <td>10</td>
+      <td>0b239_00003</td>
+      <td>5b401e15ab614332b631d552603a8d77</td>
+      <td>2022-11-30_17-40-28</td>
+      <td>...</td>
+      <td>ip-172-31-43-110</td>
+      <td>172.31.43.110</td>
+      <td>10.242707</td>
+      <td>0</td>
+      <td>10</td>
+      <td>0.003809</td>
+      <td>10</td>
+      <td>17.982020</td>
+      <td>-27.867871</td>
+      <td>/home/ubuntu/ray_results/training_function_202...</td>
+    </tr>
+  </tbody>
+</table>
+<p>4 rows × 23 columns</p>
+</div>
+
+
+
+Checkpoints, metrics, and the log directory for each trial can be accessed through the `ResultGrid` output of a Tune experiment. For more information on how to interact with the returned `ResultGrid`, see {doc}`/tune/examples/tune_analyze_results`.
```

Comment:
How do you access this after the job/cluster has been terminated? Actually this was the original user question behind this ticket.